pkg/kubelet/cri/streaming/server.go

```diff
@@ -149,6 +149,8 @@ func NewServer(config Config, runtime Runtime) (Server, error) {
 		Addr:      s.config.Addr,
```

Comment:
interestingly, I don't see any in-tree use of this package outside unit tests other than the WriteError function